### PR TITLE
Improve comment

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Thanks for bringing a bit of elegance to code comments, @casperdcl! https://github.com/iterative/terraform-provider-iterative/pull/704#discussion_r1008580659

### CODE REVIEW

File: iterative/utils/analytics.go [hunk 1]
```diff
@@ -349,7 +349,7 @@ func JitsuEventPayload(action string, e error, extra map[string]interface{}) (ma
 	}
 
 	if e != nil { 
-		// Get the type of the error as a string, but not the error message itself, because it can be sensitive.
+		// Only use error type (not message) to avoid potentially sensitive information.
```
Comment (casperdcl) [inline iterative/utils/analytics.go:352]: lol

```suggestion
		// Only use error type (not message) to avoid potentially sensitive information
```

**Review by casperdcl — APPROVED** [review]

File: iterative/utils/analytics.go (hunk 1 above)
Comment (0x2b3bfa0) [reply iterative/utils/analytics.go]: [I do.](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)

Comment (casperdcl) [reply iterative/utils/analytics.go]: Feel free to revert 🙈

Comment (0x2b3bfa0) [reply iterative/utils/analytics.go]: > _Feel free to revert_

Thanks! It's not worth the hassle to my mind, but appreciate it anyway. 🙏🏼 

> _I'm not siding with you, and I don't like it, but I am happily caving in to the concept of an official spec. Do you like my full sentences?_

This “rule” is quite flexible and not anybody follows it, less yet for inline code comments; it's mostly meant to document declarations. It's just one of those acquired tastes that some Gophers develop over time. 🦦

**Pull request merged**